pkgs/applications/graphics/gimp/plugins/default.nix

```diff
@@ -123,6 +123,23 @@ in
 
     nativeBuildInputs = with pkgs; [autoreconfHook];
 
+    postUnpack = ''
```

Comment:
Can't we unvendor ffmpeg?

Reply:
I haven't tried yet. The version that's being used by gap is more than a decade old